README.md

```diff
@@ -274,6 +278,26 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
     return true
 }
 ```
+### Badge Count
+
+#### Set Up
+Klaviyo supports custom badge counts when configuring your push notification in the editor dashboard. To set up your app, so the Klaviyo SDK properly handles them:
+
+(Pre-requisite: Set up the Notification Service Extension)
+1. In XCode, select your main app target, then go to Signing & Capabilities
+2. Add an App Groups capability and click the plus in the new section to add a new App Group
+3. Pick a name based on the scheme `group.com.[MainTargetBundleId].[descriptor]`
+4. Select your Service Extension target, and add the same App Group with the same name
+5. In your app's `Info.plist`, add a new entry for `klaviyo_app_group` as a String with the App Group name
+6. In your Notification Service Extension's `Info.plist`, add a new entry for `klaviyo_app_group` as a String with the App Group name
+
+#### Autoclearing Badge Count
+
+By default, Klaviyo SDK automatically clears all badges on app open. If you want to disable this behavior, in your app's `Info.plist`, add a new entry for `klaviyo_badge_autoclearing` as a Boolean set to `NO`. You can turn this on again by setting this to `YES`.
+
+#### Handling Other Badging Sources
+
+Klaviyo SDK handles Klaviyo pushes, but if you have other sources that change the badge count, use the `KlaviyoSDK().setBadgeCount(:)` method wherever you change the badge count to keep in sync with SDK count.
```

Comment:
Does `KlaviyoSDK().setBadgeCount(:)` actually change the badge count, or does it just update the value in UserDefaults?

Reply:
both, but it only changes it if it was able to first update it in UserDefaults

Reply:
Ok. When I read what you wrote I was a little unclear about what exactly the `setBadgeCount` method would do, and whether it should be used in addition to or instead of the native methods for setting the badge count. Maybe something like what I suggest below would make it a little more clear?

```suggestion
Klaviyo SDK will automatically handle the badge count associated with Klaviyo pushes. If you need to manually update the badge count to account for other notification sources, use the `KlaviyoSDK().setBadgeCount(:)` method, which will update the badge count and keep it in sync with the Klaviyo SDK. This method should be used instead of (rather than in addition to) setting the badge count using `UNUserNotificationCenter` and/or `UIApplication` methods.
```